src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -1,22 +1,28 @@
 import { IReduxAction } from '../../utils/ReduxUtils';
 import { IDropdownOption } from './DropdownSearch';
-import { DropdownSearchActions, IOptionsDropdownSearchPayload } from './DropdownSearchActions';
+import {DropdownSearchActions, IOptionsDropdownSearchPayload} from './DropdownSearchActions';
 import * as _ from 'underscore';
 import * as s from 'underscore.string';
 import { keyCode } from '../../utils/InputUtils';
+import {FixedQueue} from '../../utils/FixedQueue';
+import {multiSelectDropdownSearchReducer} from './MultiSelectDropdownSearch/MultiSelectDropdownSearchReducer';
+import {UUID} from '../../utils/UUID';
 
 export interface IDropdownSearchState {
   id: string;
   isOpened?: boolean;
   filterText?: string;
   options?: IDropdownOption[];
+  displayedOptions?: IDropdownOption[];
+  selectedOptions?: FixedQueue<IDropdownOption>;
   selectedOption?: IDropdownOption;
   activeOption?: IDropdownOption;
   setFocusOnDropdownButton?: boolean;
 }
 
 export const defaultSelectedOption: IDropdownOption = {
-  value: 'Select an option',
+  value: UUID.generate(),
+  displayValue: 'Select an option',
```

Comment:
You have to make this available to the users so that they can localize it or change it according to their needs. This should be a prop on your top parent component with the default set to what you've set it here.